src/timer.h

```diff
@@ -54,69 +59,79 @@ typedef struct timespec hr_timer;
 
 #ifdef CLOCK_MONOTONIC_RAW
 #define BEST_MONOTONIC CLOCK_MONOTONIC_RAW
+#define HR_CLOCK          "clock_gettime(CLOCK_MONOTONIC_RAW)"
 #else
 #define BEST_MONOTONIC CLOCK_MONOTONIC
+#define HR_CLOCK          "clock_gettime(CLOCK_MONOTONIC)"
 #endif
```

Comment:
BTW, according to https://berthub.eu/articles/posts/on-linux-vdso-and-clockgettime/ `CLOCK_MONOTONIC` is a lot cheaper to query than `CLOCK_MONOTONIC_RAW`. So we might not actually prefer the `_RAW`.

Reply:
I never saw any difference in latency between them but I guess we could just drop the checks for _RAW.